helm/prometheus-rules/templates/platform/atlas/alerting-rules/deployment.management-cluster.rules.yml

```diff
@@ -132,7 +132,7 @@ spec:
         severity: page
         team: cabbage
         topic: managementcluster
-    - alert: DeploymentNotSatisfiedBigMac
+    - alert: DeploymentNotSatisfiedShield
```

Comment:
Can you also switch area: kaas label to area: platform?

Reply:
An alert with this name already exists https://github.com/giantswarm/prometheus-rules/pull/1377/files#diff-5ea599b46b0488bf1d9fdd9264ee35236e05930d4a7747105a33e4813231e768R57

Reply:
That is fine but we can improve by merging them yes